static/docs/commands-reference/destroy.md

```diff
@@ -39,15 +39,55 @@ $ echo foo > foo
 $ dvc add foo
 $ ls -a
 
-.dvc .git code.py foo foo.dvc
+.dvc .git foo foo.dvc
 
 $ dvc destroy
 
-This will destroy all information about your pipelines as well as cache in .dvc/cache.
-Are you sure you want to continue?
+This will destroy all information about your pipelines, all data files, as well as cache in .dvc/cache.
+Are you sure you want to continue? [y/n]
 yes
 
 $ ls -a
 
-.git code.py foo
+.git .gitignore foo
 ```
+
+Let's have a look what happens when the `cache` directory is set to another
```

Comment:
there should be a ### header. Check other command references that have multiple examples please.